opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -319,24 +408,11 @@ message IntDataPoint {
   // The set of labels that uniquely identify this timeseries.
   repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
 
-  // start_time_unix_nano is the last time when the aggregation value was reset
-  // to "zero". For some metric types this is ignored, see data types for more
-  // details.
-  //
-  // The aggregation value is over the time interval (start_time_unix_nano,
-  // time_unix_nano].
-  // 
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
-  // 1970.
-  //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the
-  // timestamp may be decided by the backend.
+  // StartTimeUnixNano is optional but strongly encouraged, see the
+  // the detiled comments above Metric.
   fixed64 start_time_unix_nano = 2;
 
-  // time_unix_nano is the moment when this aggregation value was reported.
-  // 
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
-  // 1970.
+  // TimeUnixNano is required, see the detailed comments above Metric.
```

Comment:
Nit: for those of us IDE users with comment-popups when hovering over code, can you leave the "Value is UNIX epoch time in nanoseconds since Jan 1 1970 UTC" here and elsewhere?

Reply:
Done.